pkg/cmd/step/create/step_create_task.go

```diff
@@ -514,7 +516,7 @@ func (o *StepCreateTaskOptions) createEffectiveProjectConfigFromOptions(tektonCl
 			o.BuildNumber = "1"
 		} else {
 			log.Logger().Debugf("generating build number...")
-			o.BuildNumber, err = tekton.GenerateNextBuildNumber(tektonClient, jxClient, ns, o.GitInfo, o.Branch, o.Duration, o.Context)
+			o.BuildNumber, err = tekton.GenerateNextBuildNumber(tektonClient, jxClient, ns, o.GitInfo, o.Branch, o.Duration, o.Context, false)
```

Comment:
so here it is always false?

Reply:
Correct - I decided not to support this for non-metapipeline.